src/utils/jwtUtils.ts

```diff
@@ -0,0 +1,26 @@
+import jwt, {SignOptions} from 'jsonwebtoken';
+import enviroment from '../consts/enviroment';
+import {Payload} from '../models/User';
+
+class JWTUtils {
+  static generateAccessToken(
+    payload: Payload,
+    secret: string = enviroment.jwtAccessTokenSecret,
+    options: SignOptions = {expiresIn: '180 days'}
+  ) {
+    return jwt.sign(payload, secret, options);
+  }
+
+  static verifyAccessToken(
```

Comment:
need to add new methods to generate `forgetPasswordAccessToken` and `verifyForgetPasswordAccessToken`